src/app/rundown-view/components/rundown/rundown.component.html

```diff
@@ -4,7 +4,7 @@
                 id="{{segment.id}}"
                 *ngIf="!segment.isHidden"
                 [segment]="segment"
-                [isRundownActive]="isRundownActive"
+                [isRundownActiveOrRehearsal]="rundown ? isActiveOrRehearsal : false"
```

Comment:
No need for the ternary here since `isActiveOrRehearsal` already returns a boolen.

My guess is that you are using the ternary to guard against the Rundown being loaded. You should just put this "guard" within the `isActiveOrRehearsal` method:
```
public get isActiveOrRehearsal(): boolean {
  if (!this.rundown) {
    return false
  }
  ... // rest of method
}
```